mkosi

```diff
@@ -3501,7 +3500,7 @@ def build_image(args: CommandLineArguments, workspace: tempfile.TemporaryDirecto
 
     tar = make_tar(args, workspace.name, run_build_script, for_cache)
 
-    return raw, tar, root_hash
+    return raw or squashfs, tar, root_hash
```

Comment:
hmm, what does this resolve to? my python-fu is apparently not good enough to understand what a "string or string" resolves to?

Reply:
The logical expressions resolve lazily to the argument that determines the outcome. I.e., A and B becomes A if A is false, and B otherwise. A or B becomes A if A is true, and B otherwise. We know that both `raw` and `squashfs` cannot be true at once, so this essentially means "pick the non-false one".

Reply:
urks, I can't say this is a particularly pretty python feature.